src/web3client/accountManager.js

```diff
@@ -31,6 +31,12 @@ class AccountManager {
 
     // add account from private key
     addAccountFromPrivateKey(privateKey) {
+
+        // add "0x" prefix if not exists to private key if not exists
+        if (!privateKey.startsWith("0x")) {
```

Comment:
what's the good reason of adding such code?

Reply:
support both formats, from mnemonic the PK is prefixed with `0x`, but eg if exported from metamask the PK is not prefixed. 

This way we can support both cases without operator intervention

Reply:
okay, fair enough.